llvm/include/llvm/Support/NVPTXAddrSpace.h

```diff
@@ -0,0 +1,33 @@
+//===---------------- AMDGPUAddrSpace.h -------------------------*- C++ -*-===//
```

Comment:
Looks like there are still some references to AMDGPU in here

Reply:
Good catch. Fixed.